### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amyblais**
#### Summary
Update the README file.

### CODE REVIEW

**Comment by amyblais** [comment]
This was edited by Chen and reviewed by Ian and Corey so I believe this is good to merge as is.

File: README.md [hunk 1]
```diff
@@ -1,47 +1,67 @@
...
+Mattermost is an open source, private cloud, Slack-alternative from [https://mattermost.org](https://mattermost.org).
 
-It's written in Golang and React and runs as a single Linux binary with MySQL or Postgres. Every month on the 16th [a new compiled version is released under an MIT license](https://www.mattermost.org/download/).
+It's written in Golang and React and runs as a single Linux binary with MySQL or PostgreSQL. Every month on the 16th [a new compiled version is released under an MIT license](https://www.mattermost.org/download/).
 
 - [Review product documentation](http://docs.mattermost.com/).
 - [Download compiled version](https://mattermost.org/download).
 
-Try out Mattermost: 
+![s](https://camo.githubusercontent.com/b091ba61ca15a5e71d9692f626af3fb95e060ba1/687474703a2f2f7777772e6d61747465726d6f73742e6f72672f77702d636f6e74656e742f75706c6f6164732f323031352f30392f32303136303331355f763231302e706e67)
```
Comment (crspeller) [inline README.md]: Can we get a better picture for here? This one is really old.

**Comment by amyblais** [comment]
@crspeller I updated the image.

**Comment by amyblais** [comment]
Sorry, also added a frame around the image.

**Pull request merged**